crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1501,8 +1501,8 @@ where
         // create a set of validators that we should sample to request input/output objects from
         let validators_to_sample =
             if request.include_input_objects || request.include_output_objects {
-                // Always at least ask 1 validator
-                let number_to_sample = std::cmp::max(1, self.committee.num_members() / 2);
+                // Sample 10% of the validators, always at least asking 1 validator
+                let number_to_sample = std::cmp::max(1, self.committee.num_members() / 10);
```

Comment:
Is 1 too small for reliability purposes? (i.e. on devnet/tests I could see that we want more than 1)

Reply:
Per @mystenmark 's comment i've just changed this to a flat 5.